client/src/app/pages/source-platforms/components/platform-applications-table.tsx

```diff
@@ -0,0 +1,221 @@
+import * as React from "react";
+import { useTranslation } from "react-i18next";
+import { Toolbar, ToolbarContent, ToolbarItem } from "@patternfly/react-core";
+import { Table, Tbody, Td, Th, Thead, Tr } from "@patternfly/react-table";
+import spacing from "@patternfly/react-styles/css/utilities/Spacing/spacing";
+import { Ref } from "@app/api/models";
+import {
+  useTableControlState,
+  useTableControlProps,
+  deserializeFilterUrlParams,
+} from "@app/hooks/table-controls";
+import { TablePersistenceKeyPrefix } from "@app/Constants";
+import {
+  ConditionalTableBody,
+  TableHeaderContentWithControls,
+  TableRowContentWithControls,
+} from "@app/components/TableControls";
+import { SimplePagination } from "@app/components/SimplePagination";
+import { FilterToolbar, FilterType } from "@app/components/FilterToolbar";
+import { useHistory } from "react-router-dom";
+import { useFetchApplications } from "@app/queries/applications";
+
+export interface IPlatformAppsTableProps {
+  platformApplications?: Ref[];
+}
+
+const PlatformAppsTable: React.FC<IPlatformAppsTableProps> = ({
+  platformApplications,
+}) => {
+  const { t } = useTranslation();
+  const history = useHistory();
+
+  const urlParams = new URLSearchParams(window.location.search);
+  const filters = urlParams.get("filters");
+  const deserializedFilterValues = deserializeFilterUrlParams({ filters });
+
+  const {
+    data: baseApplications,
+    isFetching: isFetchingApplications,
+    error: applicationsFetchError,
+  } = useFetchApplications();
```

Comment:
It would be really great if this hook accepted a filter/param to only return applications that match the platform's application refs.  That would probably need to be added later.  Best to create a new issue referencing this idea so we could do it in future.